doc/en/components/charts/chart-types-line.md

```diff
@@ -6,55 +6,95 @@ mentionedTypes: ["XamCategoryChart", "XamDataChart"]
+- The $PlatformShort$ line chart is identical to the $PlatformShort$ spline chart in all aspects except that the line connecting data points does not have spline interpolation and smoothing for improved presentation of data. 
+
+A Line Chart includes several variants based on your data or how you want to tell the correct story with your data. These include: 
+
+- Layered Area Chart 
+- Stacked Area Chart 
+- Stepped Area Chart 
+- Stepped Line Chart 
+- Polar Area Chart 
+- Polar Line Chart 
+- Stacked 100 Area Chart  
+
+There are several use cases for a Line Chart. When you: 
+
+- Have a large, high-volume data set that fits well with the chart interactions like Panning, Zooming and Drilldown 
+- Need to compare the trends over time 
+- Want to show the difference between 2 or more data series 
+- Want to show cumulative part-to-whole comparisons of distinct categories 
+- Need to show data trends for one or more categories for comparative analysis 
+- Need to visualize detailed time-series data 
+
+Line Chart Best Practices: 
+
+- Always start the Y-Axis (left or right axis) at 0 so data comparison is accurate 
+- Order time-series data  from left to right 
+- Use visual attributes like solid lines to show a series of data 
+
+Do not Use a Line Chart When: 
+
+- You have many (more than 7 or 10) series of data. Your goal is to ensure the chart is readable. 
+- Tme-series data has similar values (data over the same period), it makes overlapped lines impossible to differentiate. 
+
+Data Structure: 
+
+- The data source must be an array or a list of data items 
+- The data source must contain at least one data item  
+- All data items must contain at least one data column (string or date time)  
+- All data items must contain at least one numeric data column  ## $PlatformShort$ Line Chart Example
 
 ## $PlatformShort$ Line Chart Example
-<!-- TODO use this iframe which will point to a new sample:
-<iframe src='{environment:dvDemosBaseUrl}/charts/category-chart-type-Line' width="100%" height="100%" seamless frameBorder="0" onload="onXPlatSampleIframeContentLoaded(this);" alt="$PlatformShort$ Line Chart Example"></iframe> -->
 
-## $PlatformShort$ Simple Line Chart
-<!-- TODO show code for CategoryChart with
-- the dataSource set to multiple data sources
-- the chartType property set to Line
-- the brushes and markerOutlines properties set to same value, e.g. "red, green, blue"
-- the markerBrushes property set for "White"
-- the markerTypes property set for "Circle"
--->
+The $PlatformShort$ Line Chart is often used to show the change of value over time such as the amount of renewable electricity produced since 2009 over a ten-year period, as we have shown on the example below.
+
+<!-- <div class="sample-container loading" style="height: 500px">
+    <iframe></iframe>
+</div>
+<div class="divider--half"></div> -->
```

Comment:
https://share.goabstract.com/e674aeff-51e9-4483-9b9b-ec63c4884c51?collectionLayerId=1c167900-5ace-4afe-80e0-e12407f27254&mode=build

Reply:
And this is the data file for the topic:
[Line Chart Data.xlsx](https://github.com/IgniteUI/igniteui-xplat-docs/files/5674119/Line.Chart.Data.xlsx)